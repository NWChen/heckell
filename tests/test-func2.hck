let g: int->int;
g(x) = x + 1 end
print("\(g(-1))");
