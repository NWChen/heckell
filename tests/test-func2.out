let f: (int -> int)
f = (x,y) ->
  (
    y + x
  )
f(4, 6)
let x: int
f(x, 5)
